docs/wiki/basic-tutorials/running.md

```diff
@@ -0,0 +1,191 @@
+---
+keywords:
+- run bot
+- running bot
+- execute bot
+- deployment
+- application
+---
+
+
+# Running and Deploying your Bot
+
+If you took the time to write a bot, at some point you'll also want to run it, either for productive use or to debug it from the IDE.
+
+## :construction_worker: Running from your IDE
+
+While developing your bot, you will want to run your bot directly from the IDE in order to quickly test changes and new features. For this, create a Run/Debug Configuration in your IDE of choice with your bots main class. Remember to also add any necessary parameters and environment variables.
+
+A working Run/Debug configuration will also enable you to run your bot with a debugger. A debugger is often considered a developers most important tool, so make sure to familiarize yourself with the debugging integration for your IDE of choice.
+
+### IntelliJ IDEA
+
+This assumes your project is set up correctly, preferably with [Gradle](/wiki/getting-started/intellij-gradle), that it can be built without errors, and that it does not yet have any run/debug configurations.
+
+**1.** Locate and click the `Add Configuration...` button at the top bar next to the start button.
+
+![](./running-idea-configurations-empty.png)
+
+**2.** Another window opens. Click the `+` button in the top left and select `Application`
+
+**3.** Give a name for your configuration and select the module to use the classpath of (usually `yourproject.main`).
+
+**4.** Select your Main class. Use the `...` button to search for it or provide the fully qualified name. If it can not be found, you most likely selected the wrong module in step 3.
+
+**5.** Optional: Set command line arguments and environment variables. For the environment variables, use the button to the right of the input field for a more convenient input window.
+
+**6.** Click `Apply` to finalize the configuration, then `OK` to close the window.
+
+![](./running-idea-configurations-create.png)
+
+**7.** Select your configuration in the drop down menu and run or debug it with the buttons to the right.
+
+![](./running-idea-configurations-bar.png)
+
+### Eclipse
+
+This assumes your project is set up correctly, that it can be built without errors, and that it does not yet have any run/debug configurations.
+
+**1.** In the menu bar, click "Run" then "Run Configurations...".
+
+**2.** In the freshly opened window, select "Java Application" on the left side, then click the leftmost button in the row above the tree view. A new configuration will appear.
+
+![](./running-eclipse-configurations-empty.png)
+
+**3.** Give a name to your configuration.
+
+**4.** Set the project and the main class. Use the "Browse..." and "Search..." buttons for an easier means of selection.
```

Comment:
Maybe something like this to avoid the awkward phrasing of `means of selection`?
```suggestion
**4.** Set the project and the main class. To easily select it, use the "Browse..." and "Search..." buttons.
```